dev/update

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Would suggest a comment here or in the readme of what this is for if it's important to the release

Reply:
Moved it to devtools directory and gave it a more explicit name